### ISSUE TITLE
Inference mistake with nn.BatchNorm3D

### ISSUE THREAD

**Issue by ruiann** [openvinotoolkit/openvino#1275]
PyTorch 1.5.1 and  openvino 2020.03 in latest MacOSX.

Sample model is simple as
```
self.conv1 = nn.Conv3d(3, 64, kernel_size=(1, 7, 7), stride=(1, 2, 2), padding=(0, 3, 3), bias=False)
self.bn1 = nn.BatchNorm3d(64)
```

The result is same between pytorch and openvino if only use conv3d like `self.conv1(x)`, but different when `self.bn1(self.conv1(x))`.

**Comment by jgespino** [comment]
Hi @ruiann 

Would it be possible to share your model and expected output for me to reproduce? Could you try converting the model once more with the model optimizer and passing the **--disable_fusing** flag? 

Regards,
Jesus

**Comment by ruiann** [comment]
I found that I forget to use model.eval() in the inference code of pytorch.
Sorry for my mistake.
